service/src/test/resources/application-app-test.yml

```diff
@@ -2,9 +2,9 @@ workspace:
   application:
     test-app-sa: Elizabeth.Shadowmoon@test.firecloud.org
     configurations:
```

Comment:
So there's no yaml schema from what I've seen. How does it know to build a map from this structure?

Reply:
See [Inject a Map from a YAML File with Spring](https://www.baeldung.com/spring-yaml-inject-map)